nekoyume/Assets/_Scripts/UI/Model/Inventory.cs

```diff
@@ -117,6 +127,45 @@ public void AddItem(ItemBase itemBase, int count = 1)
                 var blockIndex = Game.Game.instance.Agent?.BlockIndex ?? -1;
                 if (tradableItem.RequiredBlockIndex > blockIndex)
                 {
+                    if (Game.Game.instance.Agent.BlockIndex >
+                        Game.Game.instance.TempExpiredBlockIndex)
+                        return;
+
+                    var agentAddress = Game.Game.instance.Agent.Address;
+                    if (!Game.Game.instance.LegacyShopProducts.Products.ContainsKey(agentAddress))
+                        return;
+
+                    var shopItems = Game.Game.instance.LegacyShopProducts.Products[agentAddress];
+                    if (!shopItems.Exists(x => x.ProductId == tradableItem.TradableId))
```

Comment:
this case always false because productId is not TradableItem.TradableId. check shopItems inner item(itemUsable, costume, tradableFungibleItem)s TradableId,